### PR TITLE
feat: expose new vars

### PR DESCRIPTION
**Pull request by jor2**
### Description

Expose new security group variables for `terraform-ibm-base-ocp-vpc`

https://github.com/terraform-ibm-modules/terraform-ibm-ocp-all-inclusive/issues/241

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

##### Release notes content

Expose new security group variables for `terraform-ibm-base-ocp-vpc`

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Review by ocofaigh — CHANGES_REQUESTED** [review]
@jor2 can you resolve conflicts

File: version.tf [hunk 1]
```diff
@@ -4,7 +4,7 @@ terraform {
     # tflint-ignore: terraform_unused_required_providers
     ibm = {
       source  = "ibm-cloud/ibm"
-      version = ">= 1.60.0, < 2.0.0"
+      version = ">= 1.6.0, < 2.0.0"
```
Comment (ocofaigh) [inline version.tf:7]: I think you meant this right? `">= 1.62.0, < 2.0.0"`

File: examples/end-to-end-example/version.tf [hunk 2]
```diff
@@ -3,7 +3,7 @@ terraform {
   required_providers {
     ibm = {
       source  = "IBM-Cloud/ibm"
-      version = ">= 1.60.0"
+      version = ">= 1.6.0"
```
Comment (ocofaigh) [inline examples/end-to-end-example/version.tf:6]: did you mean `>= 1.62.0` ?

**Review by ocofaigh — CHANGES_REQUESTED** [review]
@jor2 I think you have some typos

**Comment by jor2** [comment]
> @jor2 I think you have some typos

I'm going to have to update all the other modules, is there an easier way?

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**